### PR TITLE
CON-2893 Markdown Builder Blueprint Exercise

### PR DESCRIPTION
**Pull request by zanninso**
[CON-2893](https://01talent.atlassian.net/browse/CON-2893)

[CON-2893]: https://01talent.atlassian.net/browse/CON-2893?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: subjects/java/checkpoints/builder-blueprint/README.md
Comment (nprimo) [inline subjects/java/checkpoints/builder-blueprint/README.md:1]: This file needs to be formatted

File: subjects/java/checkpoints/builder-blueprint/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+## Builder Blueprint
+
+### Instructions
```
Comment (nprimo) [inline subjects/java/checkpoints/builder-blueprint/README.md:3]: @zanninso It's not at all clear to me what it is supposed to do in this exercise: could you please elaborate a bit more for me? :sweat_smile:

**Review by nprimo — CHANGES_REQUESTED** [review]

File: subjects/java/checkpoints/builder-blueprint/README.md
Comment (zanninso) [reply subjects/java/checkpoints/builder-blueprint/README.md:1]: done

File: subjects/java/checkpoints/builder-blueprint/README.md (hunk 1 above)
Comment (zanninso) [reply subjects/java/checkpoints/builder-blueprint/README.md:3]: i guess I explained that enough in Discord, If you need more clarification i will be happy to explain to you here or even doing a call

**Review by nprimo — COMMENTED** [review]
@zanninso I left a small comment in the [tests PR](https://github.com/01-edu/java-tests/pull/60) - for the rest it looks ok

**Review by nprimo — APPROVED** [review]

**Pull request merged**